cinema/serializers.py

```diff
@@ -1 +1,89 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Movie, MovieSession, Actor, Genre, CinemaHall
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code. 

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)